charts/illumidesk/templates/database_cm.yaml

```diff
@@ -0,0 +1,16 @@
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: database-cm
+  namespace: {{ .Release.Namespace }}
+data:
+{{ if .Values.postgresql.enabled }}
+  database_host: {{ .Release.Name }}-postgresql.{{ .Release.Name }}.svc.cluster.local
+{{ else }}
+  database_host: .Values.externaldb.host
+{{ end }}
+  database: .Values.externalDatabase.database
```

Comment:
@Abhi94N in this case it looks like the host changes when `postgres.enabled` or when it's an `externaldb`. However the rest of the values (database, user, password, port) are set with the externaldb only. Since both the `postgres.enabled` and `externaldb` require all of these values, then perhaps the easiest thing to do is to rename the items below from `externaldb` to `commondb` or something similar.

What do you think?

Reply:
This comment is noted and will be moved to a different PR